[2024-05-29T17:03:14.433+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T17:03:14.603+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_1.task_1 scheduled__2024-03-17T00:00:00+00:00 [queued]>
[2024-05-29T17:03:14.718+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_1.task_1 scheduled__2024-03-17T00:00:00+00:00 [queued]>
[2024-05-29T17:03:14.720+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-29T17:03:14.802+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_1> on 2024-03-17 00:00:00+00:00
[2024-05-29T17:03:14.839+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=345) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-29T17:03:14.856+0000] {standard_task_runner.py:63} INFO - Started process 386 to run task
[2024-05-29T17:03:14.863+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_1', 'task_1', 'scheduled__2024-03-17T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpul6uc1ci']
[2024-05-29T17:03:14.876+0000] {standard_task_runner.py:91} INFO - Job 64: Subtask task_1
[2024-05-29T17:03:15.213+0000] {task_command.py:426} INFO - Running <TaskInstance: test_1.task_1 scheduled__2024-03-17T00:00:00+00:00 [running]> on host 2832635d2b45
[2024-05-29T17:03:16.271+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_1' AIRFLOW_CTX_TASK_ID='task_1' AIRFLOW_CTX_EXECUTION_DATE='2024-03-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-17T00:00:00+00:00'
[2024-05-29T17:03:16.279+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T17:03:16.281+0000] {logging_mixin.py:188} INFO - Run function_1!
[2024-05-29T17:03:16.282+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-29T17:03:16.288+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T17:03:16.343+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_1, task_id=task_1, run_id=scheduled__2024-03-17T00:00:00+00:00, execution_date=20240317T000000, start_date=20240529T170314, end_date=20240529T170316
[2024-05-29T17:03:16.489+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T17:03:16.785+0000] {taskinstance.py:3498} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2024-05-29T17:03:16.830+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
